lib/anyplayer/version.rb

```diff
@@ -1,3 +1,3 @@
 module Anyplayer
-  VERSION = "1.1.5"
+  VERSION = '1.1.5'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.